itou/www/apply/tests/tests_process.py

```diff
@@ -537,14 +539,14 @@ def test_cancel(self):
         self.assertTrue(job_application.state.is_cancelled)
 
     def test_cannot_cancel(self):
-        cancellation_period_end = timezone.localdate() - relativedelta(
-            days=JobApplication.CANCELLATION_DAYS_AFTER_HIRING_STARTED
-        )
         job_application = JobApplicationWithApprovalFactory(
             state=JobApplicationWorkflow.STATE_ACCEPTED,
-            hiring_start_at=(cancellation_period_end - relativedelta(days=1)),
+            hiring_start_at=(timezone.localdate()) + relativedelta(days=1),
```

Comment:
```suggestion
            hiring_start_at=timezone.localdate() + relativedelta(days=1),
```

Ouais… si tu fais ça je ne suis plus vraiment convaincu de l'intérêt de conserver ces parenthèses ?

Reply:
Black et formattage plus automatique à cause d'un bug de Neovim. Du coup, je risque d'en avoir d'autres. Merci